packages/cashier/src/cashier-providers.tsx

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import { StoreProvider } from '@deriv/stores';
+import { ThemeProvider } from '@deriv/ui';
 import { CashierStoreProvider } from './stores/useCashierStores';
 import { TRootStore } from 'Types';
 
 const CashierProviders = ({ children, store }: React.PropsWithChildren<{ store: TRootStore }>) => {
     return (
-        <StoreProvider store={store}>
-            <CashierStoreProvider>{children}</CashierStoreProvider>
-        </StoreProvider>
+        <ThemeProvider>
```

Comment:
@heorhi-deriv Since we are using the store for the `useTheme`, It's better to make the `ThemeProvider` inner child so we are sure that the `useStore` is accessible 

```ts
StoreProvider > CashierStoreProvider > ThemeProvider > children
```